src/sinks/loki/sink.rs

```diff
@@ -402,12 +415,21 @@ impl StreamSink<Event> for LokiSink {
     }
 }
 
+fn slugify_text(s: String) -> String {
+    let re = Regex::new(r"[^0-9A-Za-z_]").unwrap();
```

Comment:
I think we'd want to move this out of the function, and set it up like we've done [here](https://cs.github.com/vectordotdev/vector/blob/7cb951daed57448b80463303a07184ed537d9c54/src/template.rs?q=static+regex#L21).

I think right now we'd pay the cost of compiling the regex on every function invocation, which is pretty wasteful.

Reply:
I fixed this according to the your suggestion. Thank you!